templates/dashboards/grafana-dashboard-image-builder-worker-general.configmap.yml

```diff
@@ -1073,7 +1073,7 @@ data:
               "refId": "A"
             }
           ],
-          "title": "Error Budget Remaining",
+          "title": "Err Budget Remain ($target_duration)",
```

Comment:
Is the shortening of this because the line becomes too long? I'd argue this is less readable than before

Reply:
yes, to see which chart is affected by `$target_duration` and to fix the width of the chart title